taf/TAF_Runner.py

```diff
@@ -889,6 +894,59 @@ def audit(self):
             self.logger.info('No valid Spark Session')
             return None
 
+    # ----------------------------------------------------
+    # Should be called from the runner due to extra step
+    # ----------------------------------------------------
+    def prep_meta_table(self,
+                        pgm_name: str,
+                        step_name: str,
+                        filetyp: str = None,
+                        fl: str = None,
+                        fl2: str = None,
+                        rpt_out: str = None,
+                        clm_tbl: str = None,
+                        bsf_file_date: str = None):
+        # consider putting logging here and explain why value is needed.
+        if pgm_name is None:
+            pass
```

Comment:
not sure what this is for?

Reply:
Now, this is a valid question and I can see you're paying attention. :-) 

My hope is at some point in the near future we can go in and beef up the logging. This is supposed to be a placeholder for a check whether **pgm_name** is passed in from the notebook. It's required. I'd really like to gracefully fail to the user if they don't call the exposed API properly. In this case, pass might not be the right thing because that'll just tell the program to continue, and it'll fail anyway :-)